mmv1/products/securitycenterv2/OrganizationSccBigQueryExports.yaml

```diff
@@ -0,0 +1,145 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'OrganizationSccBigQueryExports'
+base_url: organizations/{{organization}}/locations/{{location}}/bigQueryExports
+self_link: organizations/{{organization}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}
+create_url: organizations/{{organization}}/locations/{{location}}/bigQueryExports?bigQueryExportId={{big_query_export_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'organizations/{{organization}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  post_import: templates/terraform/post_import/scc_v2_organization_big_query_export_config.go.erb
+description: |
+  A Cloud Security Command Center (Cloud SCC) Big Query Export Config.
+  It represents exporting Security Command Center data, including assets, findings, and security marks
+  using gcloud scc bqexports
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs/how-to-export-data'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.locations.bigQueryExports'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_organization_big_query_export_config_basic'
+    primary_resource_id: 'custom_big_query_export_config'
+    external_providers: ["random", "time"]
+    skip_test: true
```

Comment:
I noticed you fixed the previous error from the VCR test, should this test still be skipped?

Reply:
I can remove it and see if it works as expected.